app/services/api/v1x2/catalog/evaluate_order_process.rb

```diff
@@ -0,0 +1,68 @@
+module Api
+  module V1x2
+    module Catalog
+      class EvaluateOrderProcess
+        def initialize(order)
+          @order = order
+        end
+
+        def process
+          applicable_order_item = @order.order_items.first
+
+          relevant_order_processes = find_relevant_order_processes
+
+          # TODO: Put the order processes in order by their sequence number before
+          # doing this logic
+          @before_sequence_number = 1
+          @after_sequence_number = (relevant_order_processes.length * 2) + 1
+
+          relevant_order_processes.each do |order_process|
+            Catalog::AddToOrderViaOrderProcess.new(before_params(order_process)).process
```

Comment:
Do not make `@before_sequence_number` and `@after_sequence_number` instance variables. Rather make them local and pass them to `before_params` and `after_params` as a parameter.